e2e-tests/security-context/run

```diff
@@ -73,7 +78,7 @@ wait_backup_restore $restore
 compare_kubectl job.batch/restore-job-$restore-$cluster
 
 if [[ "${OPENSHIFT}" == 1 ]]; then
-    oc adm policy remove-scc-from-user privileged -z percona-xtradb-cluster-operator ${OPERATOR_NS:+-n $OPERATOR_NS}
+    oc adm policy remove-scc-from-user privileged -z default
```

Comment:
please remove if this line is not needed

Reply:
done